hacks_mbof/settings.py.template

```diff
@@ -91,12 +91,16 @@ WSGI_APPLICATION = 'hacks_mbof.wsgi.application'
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.mysql',
-        'NAME': 'someDatabase',
-        'USER': 'someUser',
-        'PASSWORD': 'somePassword',
-        'HOST': 'localhost',   # Or an IP Address that your DB is hosted on
-        'PORT': 'somePort',
+        #connecting to sqlite
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        #connecting to mySql
+        #'ENGINE': 'django.db.backends.mysql',
+        #'NAME': 'someDatabase',
+        #'USER': 'someUser',
+        #'PASSWORD': 'somePassword',
+        #'HOST': 'localhost',   # Or an IP Address that your DB is hosted on
+        #'PORT': 'somePort',
```

Comment:
Adding a default SQLite DB is good.  I assumed you would update the MySQL settings, since that seems to be the direction the app is taking.  Could QA testing still be done with SQLite?